integration_tests/exemplars_otel/pom.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.prometheus</groupId>
+    <artifactId>integration_tests</artifactId>
+    <version>0.10.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>exemplars_otel</artifactId>
+  <name>Integration Test - Exemplars with OpenTelemetry</name>
+
+  <properties>
+    <otel.version>1.2.0</otel.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.prometheus</groupId>
+      <artifactId>simpleclient</artifactId>
+      <version>${project.version}</version>
+      <exclusions>
+        <!-- uncomment the following to verify manually that simpleclient still works if the tracers are excluded -->
```

Comment:
Could this be another integration test?

Reply:
Good idea, I turned this into an integration test. Instead of excluding dependencies in the `pom.xml`, the new implementation of `ExemplarsOtelIT` copies the individual JARs into the Docker image so that they can be removed in the test without rebuilding the Maven project.

While I was at it, I also renamed the `tracer_...` sub-modules to `simpleclient_tracer_...`.